modules/distribution/carbon-home/conf/worker/deployment.yaml

```diff
@@ -324,7 +324,10 @@ wso2.securevault:
 wso2.datasources:
   dataSources:
     -
+      name: WSO2_CLUSTER_DB
```

Comment:
We will have to bundle WSO2_CLUSTER_DB to the pack

Reply:
Since an H2 db is defined by default, it will be created on first access